crates/solidity/outputs/cargo/slang_solidity/src/generated/language.rs

```diff
@@ -6873,6 +6873,7 @@ impl Language {
             scan_chars!(input, '\\'),
             scan_chars!(input, '\''),
             scan_chars!(input, '"'),
+            scan_chars!(input, '\r', '\n'),
```

Comment:
nit: I see we re-order the scanners to put the longer match first. I wonder if we should do that in the definition as well?

Reply:
Done in 8de0c1a009ffd991fb10536621c97e64e125955f